app/styles/templates/items/_detail.scss

```diff
@@ -16,6 +16,10 @@
     font-style: italic;
   }
 
+  .description_suggestion_container {
+    padding-bottom: 10px !important;
```

Comment:
Did we verify this on the mobile screen?

Reply:
yes. the issue happens only in mobile. web works fine.
I have confirmed the working on both web and mobile